vpp/vaapipostprocess_base.cpp

```diff
@@ -70,6 +65,28 @@ void VaapiPostProcessBase::cleanupVA()
     m_display.reset();
 }
 
+YamiStatus VaapiPostProcessBase::queryVideoProcFilterCaps(
+    VAProcFilterType filterType, void* filterCaps, uint32_t* numFilterCaps)
+{
+    if (!filterCaps)
```

Comment:
The default value for numFilterCaps is null, it's means we only have one caps, we do not care about the return value.But you are right, i need init tmp to 1 for this case.